ingests/ingests_tracker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_tracker/tracker/IngestStatesTest.scala

```diff
@@ -171,10 +171,12 @@ class IngestStatusUpdateTest
             status = updatedStatus
           )
 
-          IngestStates
+          val result = IngestStates
             .applyUpdate(ingest, update)
             .left
-            .value shouldBe a[IngestStatusGoingBackwardsError]
+            .value
+          result shouldBe a[IngestStatusGoingBackwardsError]
+          result.asInstanceOf[IngestStatusGoingBackwardsError].stored
```

Comment:
Did you mean to put an extra assertion here?

Reply:
ha, good question. I wonder what was I thinking...